sections/flyweb.tex

```diff
@@ -1,4 +1,4 @@
 \subsection{Flyweb}
 \label{sec:flyweb}
 
-FlyWeb\footnote{https://flyweb.github.io/spec/} is a Web API developed by the Mozilla Firefox community\footnote{https://wiki.mozilla.org/FlyWeb} which enables clients of Web applications to publish a local server from within the browser. Building on the concept of zero-configuration networks and its mDNS/DNS-SD protocols~\cite{rfc6762, rfc6763}, the server advertises itself in the local network and can be discovered by other devices which become clients to the server by connecting via a HTTP or WebSocket connection. This essentially enables cross-device communication within a local-area network.
\ No newline at end of file
+FlyWeb is a Web API developed by the Mozilla Firefox community which enables clients of Web applications to publish a local server from within the browser. Building on the concept of zero-configuration networks and its mDNS/DNS-SD protocols~\cite{rfc6762, rfc6763}, the server advertises itself in the local network and can be discovered by other devices which become clients to the server by connecting via a HTTP or WebSocket connection. This essentially enables cross-device communication within a local-area network.
```

Comment:
Moving footnotes to the intro as the footnotes are just necessary in the first time that we mention FlyWeb/Mozilla